src/ts/translations/de.ts

```diff
@@ -0,0 +1,41 @@
+import {Translation} from './translation-types';
+
+let lang: Translation = {
+  language: 'german',
+  settings: {
+    panel: {
+      audioQuality: 'Audioqualität',
+      videoQuality: 'Videoqualität',
+      speed: 'Geschwindigkeit',
+      audioTrack: 'Audiospur',
+    },
+    fonts: {
+      fontSize: 'Größe',
+      fontFamily: 'Schriftart',
+      fontColor: 'Farbe',
+      fontOpacity: 'Deckkraft',
+      characterEdge: 'Ränder',
+      backgroundColor: 'Hintergrundfarbe (Text)',
+      backgroundOpacity: 'Hintergrunddeckkraft (Text)',
+      windowColor: 'Hintergrundfarbe (Textbox)',
+      windowOpacity: 'Hintergrunddeckkraft (Textbox)',
+    },
+  },
+  labels: {
+    subtitles: 'Untertitel',
+    subtitleSettings: 'Subtitles settings',
+    reset: 'Zurücksetzen',
+    back: 'Zurück',
+    normal: 'Normal',
+  },
+  messages: {
```

Comment:
Why is this called `messages`, whats the differentiation to the other non-messages?